packages/ui/src/components/sign-in/steps/start.tsx

```diff
@@ -136,7 +136,7 @@ export function SignInStart() {
                   ) : null}
                 </Card.Body>
                 <Card.Actions>
-                  <Common.Loading>
+                  <Common.Loading scope='submit'>
```

Comment:
will need this added to each submit action across both sign in and signup. Not just the start step.